docker_explorer/lib/storage.py

```diff
@@ -66,21 +66,21 @@ def GetAllContainersInfo(self):
             'If it is correct, you might want to run this script'
             ' with higher privileges.').format(
                 self.container_config_filename, self.docker_directory)
-    container_info_list = [self.GetContainerInfo(x) for x in container_ids_list]
+    containers_list = [self.GetContainer(x) for x in container_ids_list]
```

Comment:
What about just returning the list comprehension?

Reply:
done